keyboards/densus/alveus/mx/info.json

```diff
@@ -0,0 +1,355 @@
+{
+    "keyboard_name": "Alveus", 
+    "manufacturer": "Mechlovin Studio",
+    "url": "", 
+    "usb": {
+        "vid": "0xDE00",
+        "pid": "0x0F70",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["B12", "B13", "B14", "A8", "A2"],
+        "cols": ["B10", "B2", "B1", "B0", "A7", "A6", "A5", "A4", "A3", "A15", "B9", "B8", "B7", "B6", "B5", "B4", "B3"]
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": true,
+        "command": true,
```

Comment:
Is there a reason these two are enabled by default?

Reply:
Hi! No particular reason for this. I always enable these on all of my PCBs. Please let me know if there are any implications that I need to consider regarding these.

Reply:
You can leave them enabled if you'd like, however they're not particularly useful for most people, increase firmware size, and can negatively impact other performance aspects such as matrix scan rate.